man/systemd.network.xml

```diff
@@ -1812,6 +1812,15 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+        <term><varname>AssignAcquiredDelegatedPrefixToken=</varname></term>
+          <listitem>
+            <para>Specifies an optional address generation mode for <varname>AssignAcquiredDelegatedPrefixAddress=</varname>.
+            When set, the lower bits of the supplied address are combined with the upper bits of a delegatad prefix received
+            from the WAN interface by the <varname>IPv6PrefixDelegation=</varname> message to form a complete address.</para>
```

Comment:
Please mention that what kind of value is taken. Also, the explanation of `AssignAcquiredDelegatedPrefixAddress=` seems to be updated.